test/benchmark/measure_performace.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
The script itself can't be run. Therefore it should neither be executable nor have a shebang line. It should probably output a message how the functions can be used when it is being sourced.